setup.cfg

```diff
@@ -0,0 +1,60 @@
+[metadata]
+name = graspologic
+version = 0.1.0
+
+description = A set of python modules for graph statistics
+long_description = file: README.md
+author = Eric Bridgeford, Jaewon Chung, Benjamin Pedigo, Bijan Varjavand
+author_email = j1c@jhu.edu
+maintainer = Dwayne Pryce
+maintainer_email = dwpryce@microsoft.com
```

Comment:
We should set up a maintainer distribution email similarly to how numpy does. Reduces single point of contact issues.

Reply:
I agree, I'm just not sure how we do that officially.